app/build.gradle

```diff
@@ -130,14 +131,17 @@ dependencies {
     implementation 'com.github.bumptech.glide:glide:4.8.0'
     kapt 'com.github.bumptech.glide:compiler:4.8.0'
 
+    implementation 'me.dm7.barcodescanner:zxing:1.9.13'
+    implementation 'com.karumi:dexter:5.0.0'
```

Comment:
Please add versions number in `build.gradle (project)` file like other dependencies, so we can manage easily all of them in one place.